extensions/algebra/complex-macros/src/lib.rs

```diff
@@ -110,6 +113,7 @@ pub fn complex_declare(input: TokenStream) -> TokenStream {
                     }
                     #[cfg(target_os = "zkvm")]
                     {
+                        Self::assert_is_setup();
```

Comment:
side comment, technically shouldn't it be `assert_is_set_up`? not worth changing though